contrib/qtxdg-tools/template.py

```diff
@@ -0,0 +1,19 @@
+pkgname = "qtxdg-tools"
+pkgver = "4.0.0"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = [
+    "cmake",
+    "ninja",
+    "pkgconf",
+    "lxqt-build-tools",
+]
+makedepends = [
+    "libqtxdg-devel",
+]
+pkgdesc = "User tools for libqtxdg"
+maintainer = "Isaac Freund <mail@isaacfreund.com>"
+license = "LGPL-2.1-or-later"
+url = "https://github.com/lxqt/qtxdg-tools"
+source = f"{url}/releases/download/{pkgver}/{pkgname}-{pkgver}.tar.xz"
```

Comment:
```suggestion
source = f"{url}/releases/download/{pkgver}/qtxdg-tools-{pkgver}.tar.xz"
```

(for everything)

Reply:
sure, the status quo did make copy pasting to create these templates easier though.